app/main.py

```diff
@@ -1,34 +1,69 @@
+from __future__ import annotations
```

Comment:
why do you need this? Seems like you use ordinary types in annotation

Reply:
Not, get_deck function  returns Deck object, but it only one place where I using it, so would it be right if I deleted this and just wrote “Deck”, it also works